mc-java community guide (#2295)
truecharts/apps: docs/apps/games/minecraft-java/Comunity_Guide.md

# Minecraft Java Community Guide

> \*Warning This guide contains information that uses Advanced/Expert settings. As a result this will be outside the scope of support!

Reviewer: ```suggestion
⚠️ **Warning This guide contains information that uses Advanced/Expert settings. As a result this will be outside the scope of support!** ⚠️
```

Reviewer: We can make it clear as day LoL

# Running Multiple MC-Java Servers

> You can easily run Multiple MC Servers. You simply need to change the external ports, Be sure not to use a unused port. There is no need to change the Minecraft port or RCON port in the server.properties

Reviewer: ```suggestion
You can easily run Multiple MC Servers. You simply need to change the external ports, Be sure to use an unused port. There is no need to change the Minecraft port or RCON port in the server.properties
```
